pkg/controllers/consolidation/controller.go

```diff
@@ -104,6 +104,7 @@ func (c *Controller) run(ctx context.Context) {
 				if err != nil {
 					logging.FromContext(ctx).Errorf("consolidating cluster, %s", err)
 				} else if result == ProcessResultNothingToDo {
+					logging.FromContext(ctx).Debugf("Attepted consolidation but nothing to consolidate")
```

Comment:
What impact does this have on the logs long term?
```suggestion
					logging.FromContext(ctx).Debugf("Attempted consolidation but nothing to consolidate")
```

Reply:
Mostly just asking what an idle karpenter looks like from a logging perspective. Does this trigger on every loop? How frequently?

Reply:
Yeah, I can imagine this would trigger too frequently. Let's discuss the logging constructs we think make sense

Reply:
Minimum of every 5 mins, I'm working on a PR to reduce the log clutter from the repeated discovering instance types, subnets, pricing, etc.